windows/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -104,18 +104,22 @@ void stopServerAction()
 	{
 		// Handle error.
 	}
-	else
-	{
-		mnuLoadBrowser->disable();
-	}
 }
 
 void loadBrowserAction()
 {
-	if (!loadBrowser("http://127.0.0.1:8080/learn"))
+	if (isServerOnline("Kolibri session", "http://127.0.0.1:8080/"))
 	{
-		// Handle error.
+		if (!loadBrowser("http://127.0.0.1:8080/learn"))
+		{
+			// Handle error.

```

Comment:
Do you know under what circumstances this would happen?

Do we have any standard way of handling errors?

Reply:
all right, thanks

since the code is just  `// Handle error.` would you mind elaborating the comment with a reference to _where_ the error is handled?